### DISCUSSION TITLE
Configuring for remote database?

### DISCUSSION THREAD

**Discussion by oguruma1218** [digitalocean/supabase-on-do#39] [Q&A]
It looks like out of the box this will install the Postgres database on the Droplet, itself. 

Isn't it recommended to host the database separately?

How can we configure this to use Digital Ocean's managed database service instead of installing the database locally?

**Comment by timothymamo** [comment]
As per [this discussion](https://github.com/digitalocean/supabase-on-do/discussions/37#discussioncomment-6047950) you will need superuser permission on your database with Supabase which is cannot be automated on the DO Managed Database.
